lib/src/screens/initial_setup/initial_setup_constants.dart

```diff
@@ -42,11 +42,11 @@ const darkBlue = Color.fromARGB(255, 7, 87, 153);
 
 /// Color variables used in initial setup screen.
 
-const kTitleTextColor = Color(0xFF30384D);
+// const kTitleTextColor = Color(0xFF30384D);
 
 /// Padding value for initial setup screen.
 
-const double kDefaultPadding = 20.0;
+// const double kDefaultPadding = 20.0;
```

Comment:
How does this affect the doc string just above? It may look somewhat odd in the generated docs?